ui/learn/src/promotion.js

```diff
@@ -37,6 +37,7 @@ function renderPromotion(ctrl, dest, pieces, color, orientation, explain) {
   if (orientation === 'white') left = 87.5 - left;
 
   var vertical = color === orientation ? 'top' : 'bottom';
+  let unbind_OnResize;
```

Comment:
snake case :thinking:

Reply:
I had it as `unbindOnResize` before but that seemed a bit confusing since it sounds like "unbind after resized" or something. But now it's gone anyway.